pins/boards.py

```diff
@@ -1123,6 +1123,12 @@ def prepare_pin_version(self, pin_dir_path, x, name: "str | None", *args, **kwar
             # TODO(compat): set display none in index.html
             context["data_preview"] = json.dumps({})
 
+        # do not show r code if not round-trip friendly
+        if meta.type in ["joblib"]:
```

Comment:
type options are "csv", "arrow", "parquet", "joblib", or "json". we will hide the R code if the pin is of `joblib` type